modules/infinite_fia/infinite_fia.module

```diff
@@ -81,11 +81,11 @@ function infinite_fia_node_view(array &$build, EntityInterface $entity, EntityVi
         GA_CD_FIA_LOGGED_IN => $datalayer_variables['environment']['isLoggedIn'] ?: '',
         GA_CD_FIA_ARTICLE_NAME => $datalayer_variables['page']['name'] ?: '',
         GA_CD_FIA_CONTENT_TYPE => $datalayer_variables['page']['contentType'] ?: '',
-        GA_CD_FIA_CONTENT_SUB_TYPE => $datalayer_variables['page']['contentSubType'] ?: '',
+        GA_CD_FIA_CONTENT_SUB_TYPE => $datalayer_variables['page']['contentSubType'] ?? '',
         GA_CD_FIA_ARTICLE_PUBLISH_DATE => $datalayer_variables['page']['articlePublishDate'] ?: '',
```

Comment:
what could possibly the reason, that the contentSubType is not set? Do other problems result from that?